frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -50,19 +90,19 @@ class AccountDetails extends React.Component<Props> {
                     <>
                       <span>
                         &uarr;
-                        {typeof account.outTransactionsCount === undefined ? (
-                          <Spinner animation="border" variant="secondary" />
+                        {transactionCount?.outTransactionsCount ? (
```

Comment:
bug: If the value will be 0, it will cast it to false and display spinner instead of "0"